apps/address-book/app/components/Panel/NewEntity.js

```diff
@@ -4,11 +4,13 @@ import { Form, FormField } from '../Form'
 import { DropDown, TextInput } from '@aragon/ui'
 import web3Utils from 'web3-utils'
 
-const ENTITY_TYPES = [ 'Individual', 'Organization', 'Project' ]
+const ENTITY_TYPES = [ 'Individual', 'Organization', 'Custom type…' ]
```

Comment:
Implementing custom types gives us a good chance to get rid of the duplicated logic we had between these two separate `ENTITY_TYPES` constants. If `ENTITY_TYPES` comes from `useAragonApi()` -> `appState`, it accomplishes that for us.